modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonadDeferLaws.kt

```diff
@@ -146,25 +143,25 @@ object MonadDeferLaws {
     df.flatMap { df }.flatMap { df }.equalUnderTheLaw(just(3), EQ) shouldBe true
   }
 
-  fun <F> MonadDefer<F>.stackSafetyOverRepeatedLeftBinds(iterations: Int = 5000, EQ: Eq<Kind<F, Int>>): Unit {
+  fun <F> MonadDefer<F>.stackSafetyOverRepeatedLeftBinds(iterations: Int = 50000, EQ: Eq<Kind<F, Int>>): Unit {
```

Comment:
5k is enough to trigger a SO, 50k makes them too slow maybe.

Reply:
Done